src/AuthModule/Components/ChangePassword/ChangePassword.tsx

```diff
@@ -1,7 +1,171 @@
-import React from 'react'
+import React, { useContext, useState } from "react";
+import { passwordValidation } from "../Validator/Validator.js";
+import logo from "../../../assets/PMS 3.svg";
+import { Link, useNavigate } from "react-router-dom";
+import axios from "axios";
+import Loader from "../../../SharedMoudule/Components/Loading/Loading";
+import { toast } from "react-toastify";
+import { baseUrl } from "../../../Constants/Components/Urls.js";
+import { useForm } from "react-hook-form";
+import { AuthContext } from "../../../Context/Components/AuthContext.tsx";
 
 export default function ChangePassword() {
+  const [showPassword, setShowPassword] = useState(true);
+  const [isLoading, setIsLoading] = useState(false);
+  const navigate = useNavigate();
+  let {  requestHeader }: any =useContext(AuthContext);
+
+  async function handleLChangePassword(values: any) {
+    try {
+      const { data } = await axios.put(`${baseUrl}/Users/ChangePassword`, values,
+      { headers: requestHeader });
+      toast.success("password changed successfully");
+      localStorage.setItem("userToken", data.token);
+      navigate("/dashboard");
+    } catch (error: any) {
+      toast.error(error?.response?.data?.message || "There's a mistake.");
+    }
+    setIsLoading(false);
+  }
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm();
+
+
+  const onSubmit = async (values: any) => {
+    setIsLoading(true);
+    handleLChangePassword(values);
+  };
+
+
+  const togglePasswordVisibility = () => {
+    setShowPassword(!showPassword);
+  };
   return (
-    <div>ChangePassword</div>
+    <>
+     <div className="auth-container vh-100  d-flex flex-column justify-content-center align-items-center">
+        <div className="logo">
+          <img className="form-logo pb-2 " src={logo} alt="logo" />
```

Comment:
You may consider adding a more descriptive `alt` for better accessibility and SEO